crates/proto/src/runtime.rs

```diff
@@ -122,6 +122,14 @@ mod tokio_runtime {
     use tokio::task::JoinSet;
     use tokio::time::timeout;
 
+    #[cfg(any(feature = "dns-over-rustls", feature = "dns-over-https-rustls"))]
+    pub const TLS_TIMEOUT: Duration = Duration::from_secs(5);
```

Comment:
I'd suggest calling these <PROTO>_HANDSHAKE_TIMEOUT

Reply:
But there are several of these, how would you name them? TLS_PROTO_HANDSHAKE_TIMEOUT and so on?